services/dashboard/app/components/Drawer.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { ReactNode } from "react";
+import { createPortal } from "react-dom";
+
+interface DrawerProps {
+  /** Controls visibility of the drawer */
+  open: boolean;
+  /** Callback fired when the user clicks on the overlay or the close button */
+  onClose: () => void;
+  /** Optional fixed width utility class (defaults to w-96) */
+  widthClass?: string;
+  /** Optional ARIA label/title for the drawer */
+  title?: string;
+  /** Drawer body */
+  children: ReactNode;
+}
```

Comment:
Hey, I don't think we need to include this comments! Can you remove them please?